server/pulp/server/webservices/views/consumers.py

```diff
@@ -0,0 +1,1082 @@
+            raise OperationPostponed(call_report)
+        else:
+            return generate_json_response_with_pulp_encoder(call_report.serialize())
+
+
+class ConsumerBindingResourceView(View):
+    """
+    Represents a specific bind resource.
+    """
+
+    @auth_required(authorization.READ)
+    def get(self, request, consumer_id, repo_id, distributor_id):
+        """
+        Fetch a specific bind object which represents a specific association
+        between a consumer and repo-distributor.
+
+        :param request: WSGI request object
+        :type request: django.core.handlers.wsgi.WSGIRequest
+        :param consumer_id: A consumer ID.
+        :type consumer_id: str
+        :param repo_id: A repo ID.
+        :type repo_id: str
+        :param distributor_id: A distributor ID.
+        :type distributor_id: str
+
+        :return: Response representing the binding
+        :rtype: django.http.HttpResponse
+        """
+
+        manager = factory.consumer_bind_manager()
+        bind = manager.get_bind(consumer_id, repo_id, distributor_id)
+        serialized_bind = serialization.binding.serialize(bind)
+        return generate_json_response_with_pulp_encoder(serialized_bind)
+
+    @auth_required(authorization.DELETE)
+    @json_body_allow_empty
+    def delete(self, request, consumer_id, repo_id, distributor_id):
+        """
+        Delete a bind association between the specified
+        consumer and repo-distributor.
+
+        :param request: WSGI request object
+        :type request: django.core.handlers.wsgi.WSGIRequest
+        :param consumer_id: A consumer ID.
+        :type consumer_id: str
+        :param repo_id: A repo ID.
+        :type repo_id: str
+        :param distributor_id: A distributor ID.
+        :type distributor_id: str
+
+        :raises OperationPostponed: will dispatch a task if 'notify_agent' is set to True
+        :raises InvalidValue: if some parameters are invalid
+
+        :return: Response representing the deleted binding(in case 'notify agent' is set to False)
+        :rtype: django.http.HttpResponse
+        """
+
+        body = request.body_as_json
+        forced = body.get('force', False)
+        if not isinstance(forced, bool):
+            raise InvalidValue(['force'])
+        options = body.get('options', {})
+        if not isinstance(options, dict):
+            raise InvalidValue(['options'])
+        if forced:
+            call_report = consumer_task.force_unbind(consumer_id, repo_id, distributor_id, options)
+        else:
+            call_report = consumer_task.unbind(consumer_id, repo_id, distributor_id, options)
+
+        if call_report.spawned_tasks:
+            raise OperationPostponed(call_report)
+        else:
+            return generate_json_response_with_pulp_encoder(call_report.serialize())
+
+
+class ConsumerContentActionView(View):
+    """
+    Views for content manipulation on the consumer.
+    """
+
+    @auth_required(authorization.CREATE)
```

Comment:
Actually git blame told that this part of pulp was implemented almost 3 years ago, so i think your suggestion about changing this to 'execute' permission is valid and logical